private/sass/addons/_lightbox.scss

```diff
@@ -1,4 +1,4 @@
-//######################################################################################################################
+// #####################################################################################################################
```

Comment:
we still use it: https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/develop/docs/guidelines/comments.rst